kernel/fablib/actions/component/stop.go

```diff
@@ -28,17 +28,25 @@ func Stop(componentSpec string) model.Action {
 	}
 }
 
+func StopInParallel(componentSpec string) model.Action {
+	return &stop{
+		componentSpec: componentSpec,
+		parallel:      true,
+	}
+}
+
```

Comment:
I don't think we need to deal with it for this PR, but we're eventually going to want an upper limit that's configurable per-installation, per-model, per-operation for parallelism... as in "start in parallel, up to this many concurrent operations".

Reply:
So rather than tracking `parallel: true` in each component, we may want `concurrency: 1` (or any other value) in those components. That probably wants to be initialized from the user's configuration and/or the label.

Reply:
I was thinking we'd want a limiter as well. Having `concurrent: N` seems like a nice way to expose that

Reply:
Added the concurrency: N options